i18n/speaking/poster-session.i18n.js

```diff
@@ -4,18 +4,17 @@ export default genI18nMessages({
     'en-us': {
         title: 'Call for Poster Session',
         intro: [
-            "Welcome to PyCon TW's first Poster Session!",
-            'Poster Session is a presentation format that allows speakers to present their topic through posters and interact directly with individual audiences. At PyCon Taiwan 2024, we tried the Poster Session activity for the first time, hoping to allow more people to introduce their results or issues of concern. You can submit a paper on any Python-related topic and it will be displayed as a poster, and then audiences can interact with you directly.',
+            'Poster Session is a presentation format that allows speakers to present their topic through posters and interact directly with individual audiences. At PyCon Taiwan 2024, we tried the Poster Session activity for the first time, hoping to allow more people to introduce their results or issues of concern.You can submit a paper on any Python - related topic and it will be displayed as a poster, and then audiences can interact with you directly.',
```

Comment:
標點符號的問題
```suggestion
            'Poster Session is a presentation format that allows speakers to present their topic through posters and interact directly with individual audiences. At PyCon Taiwan 2024, we tried the Poster Session activity for the first time, hoping to allow more people to introduce their results or issues of concern. You can submit a paper on any Python-related topic and it will be displayed as a poster, and then audiences can interact with you directly.',
```

Reply:
修改一版，如果不嫌棄的話可斟酌採納
```suggestion
            'Poster Session allows speakers to showcase their topics using posters and engage directly with individual audiences. At PyCon Taiwan 2024, we'll host the Poster Session for the first time, aiming to provide a platform for more individuals to share their projects or topics of interest. You are invited to submit a paper on any Python-related topic, which will be exhibited as a poster, enabling you to have direct conversations with attendees.',
```